### ISSUE TITLE
labels containing 'interpolate' causes 'Syntax error'

### ISSUE THREAD

**Issue by cpelley** [mermaid-js/mermaid#3812]
### Description

Labels containing 'interpolate' causes mermaid to return 'syntax error'
Potentially related to https://github.com/mermaid-js/mermaid/issues/2388 which seems similar except that refers to node id in that case.  I assume these 'reserved words' wouldn't apply to labels.

I can get things to behave as expected by using double quotes as per [mermaid docs](https://mermaid-js.github.io/mermaid/#/flowchart?id=special-characters-that-break-syntax) for 'special characters'.

Interestingly 'interpolate' causes a problem while other "reserved words" like 'classDef' and 'default' don't cause a problem.

### Steps to reproduce

See code example.
```
graph TD
    A[interpolate]
```
Live graph [here](https://mermaid.live/edit#pako:eNpFjrEOwjAQQ3-l8twvyIbUlQkmCMOpOWhEk4uO61BV_XcOGPBkW0-WN4ySGAEPpTZ15yHWznW45mqsTWYyvqFHYS2Uk4Pbh4iwiQtHBLeJ9BkR6-4cLSantY4Ipgv3WFryhSGT7xeEO80vbxvVi8g_c8omevw9-R7a3zUXM4U)

### Code Sample

```text
https://mermaid.live/edit#pako:eNpFjrEOwjAQQ3-l8twvyIbUlQkmCMOpOWhEk4uO61BV_XcOGPBkW0-WN4ySGAEPpTZ15yHWznW45mqsTWYyvqFHYS2Uk4Pbh4iwiQtHBLeJ9BkR6-4cLSantY4Ipgv3WFryhSGT7xeEO80vbxvVi8g_c8omevw9-R7a3zUXM4U
```

### Setup

**Desktop**

- OS and Version: Windows/Linux
- Browser and Version: Chrome/Firefox

**Comment by jgreywolf** [comment]
@cpelley Just make sure to enclose the text in double quotes and this will work as you expect

**Comment by cpelley** [comment]
> @cpelley Just make sure to enclose the text in double quotes and this will work as you expect

Indeed, I mentioned this in the issue description 👍 

I raised the issue as the documentation doesn't include any mention of 'interpolate' being reserved so wondered whether or not this should need escaping at all for the label.  If not, 'reserved words' could have mention within the documentation, perhaps under the 'Special characters that break syntax' section.

I discovered the issue only after bisecting my graph to isolate the specific problematic label.
I could of course be missing something as to why it should be self evident but I didn't know ahead of time the issue.

I would be happy to submit PR if a documentation change is agreed as desirable.
